src/components/FormValidator.js

```diff
@@ -63,14 +64,18 @@ export default class FormValidator {
     this._toggleButtonState();
   }
 
+  resetForms() {
```

Comment:
Needs correction: the form validator should not be responsible for resetting the form